lib/puppet/reports/servicenow.rb

```diff
@@ -6,17 +6,16 @@
   include Puppet::Util::Servicenow
 
   def process
-    decision_parameters = "status: #{status}, "\
-                          "corrective_change: #{corrective_change}, "\
-                          "noop_pending: #{noop_pending}"
-    Puppet.info(sn_log_entry("report attributes parameters: #{decision_parameters}"))
```

Comment:
I removed this message b/c `resource_statuses` can be pretty large so figured if we're not printing that, then there's no need to print the remaining parameters.